docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -166,6 +156,84 @@ support_bundle/
         └── users.json
 ```
 
+#### Universe logs
+
+Logs generated by YugabyteDB in the universe.
+
+#### Output files
+
+The `.out` files created in the YugabyteDB nodes.
+
+#### Error files
+
+The `.err` files created in the YugabyteDB nodes.
+
+#### G-Flag configurations
+
+Flags set on the universe. It is the `server.conf` file in the YugabyteDB nodes.
+
+#### Instance files
+
+Instance file in the `/master` and `/tserver` directories of YugabyteDB nodes.
+
+#### Consensus meta files
+
+Consensus-meta folder in the `/master` and `/tserver` directories of YugabyteDB nodes.
+
+#### Tablet meta files
+
+Tablet-meta folder in the `/master` and `/tserver` directories of YugabyteDB nodes.
+
+#### Node agent logs
+
+Node agent logs generated in the YugabyteDB nodes (if node agent is enabled).
+
+#### Core Files
+
+The Core Files component collects all the core files and mitigates two issues that may happen when collecting files such as the following:
+
+- Files can be very large (for example, 200GB+)
+- There can be a huge number of files (for example, when a crashloop happens)
+
+When you create a support bundle, if you select the Core Files component (by default, the component is selected), there are two optional Core files properties:
+
+- Maximum number of recent core files: YugabyteDB Anywhere (YBA) collects the most recent "N" number of files. "N" is set to a default value of collecting 1 file.
+
+- Maximum core file size: This field collects the recent core files only if the core file size is below the specified size limit. It is set to a default value of 25000000000 bytes (25GB).
+
+YBA also provides a runtime flag `yb.support_bundle.allow_cores_collection`, which is used to globally disable cores collection across any new support bundles generated on the platform. This flag can only be set by the SuperAdmin and is true by default.
+
+#### YB-Controller logs
+
+YBC logs generated in `/controller/logs` folder in the YugabyteDB nodes (if YB-Controller is enabled).
+
+#### Kubernetes info
+
+This component exclusively includes metadata for Kubernetes-based universes. The detailed metadata list is as follows:
```

Comment:
We should also add a note here saying that these below files are only collected by YBA if we have sufficient permissions to get this info. Else the file collection is skipped.